docs/data/material/components/snackbars/snackbars.md

```diff
@@ -9,109 +9,88 @@ waiAria: https://www.w3.org/TR/wai-aria-1.1/#alert
 
 # Snackbar
 
-<p class="description">Snackbars provide brief notifications. The component is also known as a toast.</p>
-
-Snackbars inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn't interrupt the user experience, and they don't require user input to disappear.
-
-Snackbars contain a single line of text directly related to the operation performed.
-They may contain a text action, but no icons. You can use them to display notifications.
+<p class="description">Also known as a toast, snackbars are used for brief notifications of a process that has been performed or will be performed.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-**Frequency**: Only one snackbar may be displayed at a time.
+## Introduction
 
-## Simple snackbars
+```jsx
+import Snackbar from '@mui/material/Snackbar';
+```
 
-A basic snackbar that aims to reproduce Google Keep's snackbar behavior.
+The Snackbar provides users updates on an app's processes, temporarily appearing on the screen.
+Inspired by Google Keep, this demo shows a basic snackbar with a text element and two actions.
 
 {{"demo": "SimpleSnackbar.js"}}
 
```

Comment:
How about a note on usage here? I think it's helpful to compare/contrast with Alerts and Dialogs so users can understand the appropriate use cases.

```suggestion
:::info
Snackbars differ from [Alerts](/material-ui/react-alert/) in that Snackbars have a fixed position and a high z-index, so they're intended to break out of the document flow; Alerts, on the other hand, are usually part of the flow—except when they're [used as children of a Snackbar](#use-with-alert).

Snackbars also from differ from [Dialogs](material-ui/react-dialog) in that Snackbars are not intended to convey _critical_ information or block the user from interacting with the rest of the app; Dialogs, by contrast, require input from the user before they can be dismissed.
:::

```